### DISCUSSION TITLE
Display some informations on the oled or tft screen

### DISCUSSION THREAD

**Discussion by Sherhang** [ExpressLRS/ExpressLRS#1802] [Q&A]
Can it display some informations such as gps、rssi dbm on the oled or tft screen?

**Comment by SunjunKim** [comment]
Unfortunately, the displays are unsuitable for frequent dynamic information updates due to timing constraints.

**Reply by commanderguy3001** [reply]
could we do it on the second core on the ESP?

**Reply by SunjunKim** [reply]
I think it cannot be working in that way with the current hardwares.

**Reply by pkendall64** [reply]
The display on the ESP32 is already on a different core. The main problem is that we don't process the telemetry, we just send it back to the handset.

**Reply by commanderguy3001** [reply]
so there is no hardware-reason that prevents us (be that flash size, cpu compute power, ram constraints or IO stuff) from doing it, but only that we don't wanna have to actually work with the data for simplicity of the code?

**Reply by pkendall64** [reply]
Well, I'm not sure thats the only reason. I mean, who's going to flip their radio over to look at that while they're flying when they have a honking great screen on the front of the radio!?

**Reply by SunjunKim** [reply]
> who's going to flip their radio over to look at that while they're flying

LR flyers with Futaba Radio will probably do this

**Reply by Sherhang** [reply]
I think it will be useful when the quard crashe. Gps info helps to find the quard，especiall for closed-source ratio like Futaba/Raidolink At9s Pro.
